### DISCUSSION TITLE
popup windows or save windows

### DISCUSSION THREAD

**Discussion by AnthonyBAC** [mylinuxforwork/dotfiles#185] [General]
Pop-up windows open as a positioned window and not as floating windows, is there a way to configure that? or why is it happening? Also some programs like netbeans open as floating when it should not be like that, can anyone help me? or when im gonna put images as example 

![image](https://github.com/user-attachments/assets/49c3333a-b4a8-42cd-82b4-1686a7d80a77)

when i want to open a new project on obsidian it opens like that not in floating, the same happens when a popup windows open in my browser

**Comment by mylinuxforwork** [comment]
You can define a special window rule for Obsidian.
Get the class with hyprctl clients
And create a window rule in custom.conf

**Reply by AnthonyBAC** [reply]
Ty :)
